packages/core/src/routes/session.ts

```diff
@@ -67,13 +67,14 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
       }),
     }),
     async (ctx, next) => {
+      const { jti } = await provider.interactionDetails(ctx.req, ctx.res);
       const { username, password } = ctx.guard.body;
-      ctx.log.type = LogType.SignInUsernamePassword;
-      ctx.log.username = username;
+      const type = 'SignInUsernamePassword';
```

Comment:
nit: since we already have type well defined, no need to define a variable for it, just make it inline should be fine.

Reply:
If we need to change the type here, it will be a little harder to modify the inlined string literal.

e.g. inlined `SignInSocial` 
![image](https://user-images.githubusercontent.com/10594507/165021101-19ae317d-52e4-4d21-ad13-fcdbedafc08a.png)